hcn/hcnnetwork.go

```diff
@@ -310,6 +334,56 @@ func (network *HostComputeNetwork) Delete() (*HostComputeNetwork, error) {
 	return nil, nil
 }
 
+// ModifyNetworkSettings updates the Policy for a network.
+func (network *HostComputeNetwork) ModifyNetworkSettings(request *ModifyNetworkSettingRequest) error {
```

Comment:
Dont you want to check `.Settings != nil`?

Reply:
Settings can be nil and we will omit if empty.